BuildScript/Main/Remotion.build

```diff
@@ -25,13 +25,17 @@
     <ExtensionTasksPath>$(MSBuildExtensionPackPath)</ExtensionTasksPath>
     <DependDBBuildProcessorToolPath Condition="'$(NuGetToolPath)' == ''">$(PackagesDirectory)DependDB.BuildProcessor.2.0.0\tools\</DependDBBuildProcessorToolPath>
     <DependDBNuGetPreProcessorToolPath Condition="'$(NuGetToolPath)' == ''">$(PackagesDirectory)DependDB.BuildProcessor.NuGetPreProcessor.2.0.0\tools\</DependDBNuGetPreProcessorToolPath>
-    <NUnitToolPath Condition="'$(NUnitToolPath)' == ''">$(PackagesDirectory)NUnit.Runners.2.6.3\tools\</NUnitToolPath>
+    <NUnitToolPath Condition="'$(NUnitToolPath)' == ''">$(PackagesDirectory)NUnit.ConsoleRunner.3.11.1\tools\</NUnitToolPath>
     <NuGetToolPath Condition="'$(NuGetToolPath)' == ''">$(PackagesDirectory)NuGet.CommandLine.3.5.0\tools\</NuGetToolPath>
     <NuGetForMSBuildPath Condition="'$(NuGetForMSBuildPath)' == ''">$(PackagesDirectory)NuGet.for.MSBuild.1.4.3\build\</NuGetForMSBuildPath>
     <RemotionBuildTasksPath Condition="'$(RemotionBuildTasksPath)' == ''">$(PackagesDirectory)Remotion.BuildTools.MSBuildTasks.1.0.6862.28795\tools\</RemotionBuildTasksPath>
     <RemotionBuildScriptPath Condition="'$(RemotionBuildScriptPath)' == ''">$(PackagesDirectory)Remotion.BuildScript.2.0.0-alpha020\BuildTargets\</RemotionBuildScriptPath>
   </PropertyGroup>
 
+  <ItemGroup>
+    <NUnitExtensionsPaths Include="$(PackagesDirectory)NUnit.Extension.NUnitV2Driver.3.8.0\"/>
```

Comment:
We want to do an opt-in to the backward-compatible build, correct. Or do we want to, for now, deliver the Nunit2-compat to all projects regardless of their nunit2 requirements?

Reply:
I remember you saying we can always ship the NUnit2 compatibility if having it enabled doesn't affect running V3 tests, and it doesn't so I added it there. I could remove it though, I don't think it matters much. I also decided against always shipping [NUnit.Extension.TeamCityEventListener](https://www.nuget.org/packages/NUnit.Extension.TeamCityEventListener/) since it wasn't "essential" to the builld, even though we'd probably want it in every project

Reply:
that nunit-compat question was in regard to "can we do this, if we have to". But if we don't need to always include it, it's one moving part less we have to worry about.

The teamcity-one: will need to have more information to make a good descison on include or not